src/publish/confluence/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const DELETE_DISABLED = false;
 
-export const EXIT_ON_ERROR = true;
+export const EXIT_ON_ERROR = false;
```

Comment:
I'm not super well informed about what this means, but I generally vote for failing hard / exit on error by default so we find any unexpected errors quickly / don't proceed with operations once the outcome / state become indeterminant. This might be an expected error or something, so feel free to ignore peanut gallery if not helpful.

Reply:
It is worth thinking about.  This is mainly for my testing today.  In production, it seems wiser to fail hard rather than let run-away operations go.  It is a tough call actually.  I need to think more on this.  Thank you for raising it.